Source/Core/EasingFunction.js

```diff
@@ -22,21 +22,21 @@ var EasingFunction = {
    * @type {EasingFunction.Callback}
    * @constant
    */
-  QUADRACTIC_IN: Tween.Easing.Quadratic.In,
+  QUADRATIC_IN: Tween.Easing.Quadratic.In,
```

Comment:
@benjaminabb can you back out the changes to this file into a separate PR?  There are a few more changes we'll need to make regarding fixing this typo before it can be merged into master